src/app/profile/profile.component.css

```diff
@@ -24,3 +24,8 @@ img {
   border-radius: 25px;
   height: 200px;
 }
+
+.item-card {
```

Comment:
I feel like this class should belong to the `item-card.component.css` file if you need padding. I don't feel like this style is specific to the profile component.